index.html

```diff
@@ -421,14 +516,7 @@ <h2>
           <ol class="algorithm">
             <li>Set the [=assigned launch consumer=] to |consumer|.
             </li>
-            <li>[=list/For each=] |launch_params:LaunchParams| of [=unconsumed
-            launch params=]:
-              <ol>
-                <li>Invoke |consumer| with |launch_params|.
-                </li>
-              </ol>
-            </li>
-            <li>Set [=unconsumed launch params=] to the empty [=list=].
+            <li>Run the steps to [=process unconsumed launch params=].
```

Comment:
Same here.

Reply:
I'm not sure it makes sense here since this is in the context of a LaunchQueue method.